ReactNativeClient/lib/shim.js

```diff
@@ -161,6 +161,24 @@ shim.uploadBlob = () => {
 	throw new Error('Not implemented');
 };
 
+shim.uploadChunk = async function(url, handle, options) {
+
```

Comment:
Please make a copy of the options if you are going to modify them: `options = Object.assign({}, options)`